packages/psss-server/src/connections/AuthConnection.ts

```diff
@@ -24,11 +24,11 @@ export class AuthConnection extends ApiConnection {
     super(basicAuthHandler);
   }
 
-  async login({ emailAddress, password }: Credentials) {
+  async login({ emailAddress, password, deviceName }: Credentials) {
     const response = await this.post(
       'auth',
       { grantType: 'password' },
-      { emailAddress, password, deviceName: 'PSSS Server' },
+      { emailAddress, password, deviceName: `PSSS-SERVER: ${deviceName}` },
```

Comment:
I assumed that we still want the deviceName captured but let me know if not and i can remove it.